src/main/java/io/nats/client/impl/NatsConnection.java

```diff
@@ -1516,7 +1517,11 @@ public Duration RTT() throws IOException {
         catch (ExecutionException e) {
             throw new IOException(e.getCause());
         }
-        catch (InterruptedException | TimeoutException e) {
+        catch (TimeoutException e) {

```

Comment:
Recombine the execution and the timeout. I know it makes sense to get the cause of the ExecutionException, but that might change what someone has already coded for.